src/heroes/sombra/translocator_cooldown_reset.opy

```diff
@@ -0,0 +1,49 @@
+#Global variables
+
+globalvar STEALTH_REFUND_FRACTION 0
+globalvar NEGATIVE_STATUS_PREVENTS_REFUND 1
+globalvar MINIMUM_TRANSLOCATOR_COOLDOWN 2
+
+
+#Player variables
+
+playervar stealth_timer 0
+playervar stealth_max_Cooldown 1
+playervar ult_charge_Percent_cache 2
```

Comment:
Isn't this unused at this point?

Reply:
so reading again apparently the ultimate cache is so is if sombra presses Q while actually having ult the game knows she code knows she cancelled stealth. so  i need to add back the code i deleted and make it only run for sombra because we don't need anyone elses ult charge

Reply:
so yeah you are right no reason to cache ult charge. removing it now that i know how.